app/models/group.rb

```diff
@@ -1,3 +1,12 @@
 class Group < ApplicationRecord
   default_scope { order(:name) }
+
+  has_many :group_assignments, class_name: GroupAssignment
```

Comment:
if you are using `class_name: ` it should be a string to avoid load order issues.